angular/projects/spark-angular/src/lib/components/sprk-promo/sprk-promo.stories.ts

```diff
@@ -86,21 +119,49 @@ flag.story = {
 export const withImage = () => ({
   moduleMetadata: modules,
   template: `
-    <sprk-promo
-      title="Title"
-      subtitle="Subtitle"
-      media="img"
-      imgAlt="placeholder"
-      imgSrc="https://spark-assets.netlify.app/house.jpg"
-      cta="button"
-      ctaText="Learn More"
-      buttonHref="#"
-      idString="promo-3"
-      hasBorder="true"
-    >
-      Lorem ipsum dolor. Sit amet pede. Tempus donec et. Suspendisse id
-      inventore integer eum non enim diam habitant. Maecenas nunc per lacus
-      neque egestas. Diam quod curabitur.
+    <sprk-promo idString="image-promo" hasBorder="true" additionalClasses="sprk-o-Stack--split@s">
+      <a href="#nogo" sprkStackItem class="sprk-o-Stack__item--half@s">
+        <img
+          sprkPromoMedia
+          alt="A house and front yard."
```

Comment:
Good catch! I'm not sure what the alt text should be here since the link doesn't go anywhere, but the alt text should be describing the link behavior.

Reply:
Olya said to make it Additional insights about this property.